.github/workflows/kind-e2e.yaml

```diff
@@ -166,6 +169,32 @@ jobs:
 
         kind create cluster --config kind.yaml
 
+    - name: Configure NetworkPolicy
```

Comment:
not very important, but I'm wondering why this is split out and done early rather than being done as part of Apply NetworkPolicy?